changelog/unreleased/issue-3572

```diff
@@ -0,0 +1,7 @@
+Enhancement: Restore ownership by name
+
+Restic now optionally restores file ownership on UNIX systems by the names of the user and group.
```

Comment:
Please mention the necessary parameter and that it is for the `restore` command.